frontend/packages/component-library/src/textField/textfield.module.scss

```
@@ -5,225 +5,118 @@
 
 // TextField common styles
 .textField {
-  display: inline-flex;
-  flex-direction: column;
-  align-items: flex-start;
-  gap: 0.125rem;
-  min-width: 18.75rem;
-  margin: 0;
-
-  input {
-    margin-bottom: 0;
-    border-radius: 0.25rem;
-    border-width: 1px;
-    border-style: solid;
-    caret-color: var(--borders-brand-teal-primary);
-    box-sizing: border-box;
-    height: unset;
-
-    &:disabled,
-    &:read-only {
-      cursor: not-allowed;
-    }
-
-    &:focus-visible {
-      outline: 2px solid var(--borders-brand-teal-primary);
-      outline-offset: 2px;
-      box-shadow: none;
-    }
+  margin-bottom: 0;
+  border-radius: 0.25rem;
+  border-width: 1px;
+  border-style: solid;
+  caret-color: var(--borders-brand-teal-primary);
+  box-sizing: border-box;
+  height: unset;
+
+  &:disabled,
+  &:read-only {
+    cursor: not-allowed;
   }
 
-  .textFieldHelperSection {
-    display: flex;
-    align-items: baseline;
-    align-self: stretch;
+  &:focus-visible {
+    outline: 2px solid var(--borders-brand-teal-primary);
+    outline-offset: 2px;
+    box-shadow: none;
```

You can use [this new mixin](https://github.com/code-dot-org/code-dot-org/blob/95527bb6a862b0aa8eb2f753b98b8d33dba242cc/frontend/packages/component-library-styles/mixins.scss#L155-L159) here 🤩 

Related Q, @levadadenys do we need to add `box-shadow: none;` to that mixin, or is this specific to this component?

```suggestion
    @include focus-styles
```

---

Good question. I don't remember for sure, but I think we can try adding `box-shadow: none;` into that mixin and see if that will break anything 😄